### DISCUSSION TITLE
How I fix this?

### DISCUSSION THREAD

**Discussion by Pr0Dr1v4rBR** [otomad/OtomadHelper#59] [Q&A] [answered]
![Captura de tela 2023-12-27 023538](https://github.com/otomad/OtomadHelper/assets/154955473/b86d4864-8cfe-4681-ba61-438a2d714a26)

(My Sony Vegas is in Brazilian Portuguese)

**Comment by Evauation** [comment]
@otomad 

~Vegas Pro 15 will be missing some api features that the script can't support, quoting from old patch notes.~
~Elastique & Classic pitch change. Change tuning method. Some PV rhythm visual effects and Reserve Formant, among others.~

~My only suggestion is to have the Audio setting turned off.~

**Comment by otomad** [comment, marked as answer]
No, it is duplicate of #24. @Evauation @Pr0Dr1v4rBR 

You're using the Vegas Pro 16+ Version of Otomad Helper on Vegas Pro 15.

This is what you should be using: 
https://github.com/otomad/OtomadHelper/releases/download/v4.31.28.0/1_otomad_helper_v4.31.28.0_vegas1415.zip
